cloudbuild_release.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is meant to be used by the Build Trigger set in Cloud Build for auto
+# release. This is what it does:
+# - Builds an image using the last GitHub commit hash as the tag.
+# - Pushes that image to Container Registry.
+# - Runs all integration test from that very image.
+# - Pushes the verified image to the main project (gcp-variant-transforms).
+
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'build'
+      - '--tag=gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
+      - '--file=docker/Dockerfile'
+      - '.'
+    id: 'build-gcp-variant-transforms-docker'
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'push'
+      - 'gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
+    id: 'push-gcp-variant-transforms-docker'
+  - name: 'gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
+    args:
+      - 'bash'
+      - './deploy_and_run_tests.sh'
+      - '--keep_image'
+      - '--skip_build'
+      - '--project ${PROJECT_ID}'
+      - '--image_tag ${COMMIT_SHA}'
+      - '--run_unit_tests'
+      - '--run_preprocessor_tests'
+      - '--run_bq_to_vcf_tests'
+      - '--run_all_tests'
+      - '--test_name_prefix cloud-ci-'
+    id: 'test-gcp-variant-transforms-docker'
+  - name: 'gcr.io/cloud-builders/docker'
```

Comment:
You shouldn't need this step: we will be running it in the gcp-variant-transforms project, and if we aren't we shouldn't cross tag

Reply:
OK you've convinced me to leave this as is :)

Let's change the step ids to something like

build-image
deploy-to-test-project
test-image
deploy-to-serving-project

so it's at least clear the difference?